src/main/java/com/ridemart/util/JwtUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.ridemart.util;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.stereotype.Component;
+
+import java.security.Key;
+import java.util.Date;
+
+@Component
+public class JwtUtil {
+
+    private static final Key SECRET_KEY = Keys.secretKeyFor(SignatureAlgorithm.HS256);
+    private static final long EXPIRATION_TIME = 1000 * 60 * 60;
```

Comment:
variables like this need to be managed in the application.properties file and then injected using the @Value annotation or using the @Configuration annotation to create a whole config file that reads the app properties and exposes some getter methods to use the properties from app properties
at this stage, use @Value, it is simpler and faster